change/@office-iss-react-native-win32-5e79c3d4-72cf-454b-b113-878d1f3d5c66.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Allow native module to set JS-side QuirkSettings",
```

Comment:
Only thinking about the term `quirks` for me it evokes the idea of backwards compat settings. i.e. IE6 Quirks mode...
Especially feels strange if the DefaultQuirks have both flags set to true...

Is that the intended use of these flags? If not perhaps a more neutral term like 'Flight' or 'Experiments' which I've seen used more widely for the general purposeness might increase its usage.

Reply:
I named it this to match the native MSRN `QuirkSettings` interface, which allows runtime flags to change behavior in native. I was thinking if we were to move this to the OSS version, we could pipe constants from there so they can be set before instance init.